content/en/tutorials/reproducible-machine-learning-cloud-pods/index.md

```diff
@@ -0,0 +1,230 @@
+
+{{< command >}}
+$ awslocal lambda create-function --function-name reproducible-ml-sample \
+  --runtime python3.8 \
+  --role r1 \
+  --handler train.handler \
+  --timeout 600 \
+  --code '{"S3Bucket":"reproducible-ml","S3Key":"lambda.zip"}' \
+  --layers arn:aws:lambda:us-east-1:446751924810:layer:python-3-8-scikit-learn-0-23-1:2
+{{< / command >}}
+
+We have specified the function name as `reproducible-ml-sample`, the runtime as `python3.8`, the handler function as `train.handler`, and the S3 bucket and key where the `zip` file is located. We have also specified the `python-3-8-scikit-learn-0-23-1` layer to be used by the Lambda function. This layer contains the `scikit-learn` library and its dependencies. We can now invoke the Lambda function using the `awslocal` CLI:
+
+{{< command >}}
+$ awslocal lambda invoke --function-name reproducible-ml-sample /tmp/test.tmp
+{{< / command >}}
+
+The logs of the Lambda invocation should be visible in the LocalStack container output (with `DEBUG=1` enabled):
+
+```bash
+null
+>START RequestId: 65dc894d-25e0-168e-dea1-a3e8bfdb563b Version: $LATEST
+> --> prediction result: [8 8 4 9 0 8 9 8 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 9 6 7 8 9
+...
+...
+>  9 5 4 8 8 4 9 0 8 9 8]
+> END RequestId: 6...
+```
+
+## Creating a Cloud Pod
+
+After deploying the Lambda function, we can now create a Cloud Pod to share our local infrastructure & instance state with other LocalStack users in your organization. To save the state of our LocalStack instance, we will use the `save` command:
+
+{{< command >}}
+$ localstack pod save reproducible-ml
+
+Cloud Pod reproducible-ml successfully created
+{{< / command >}}
+
+{{< alert title="Saving Cloud Pods locally" >}}
+You can also export a Cloud Pod locally by specifying a file URI as an argument. To export on a local path, run the following command:
+
+{{< command >}}
+$ localstack pod save file://<path_on_disk>/<pod_name>
+{{< / command >}}
+
+The output of the above command will be a `<pod_name>.zip` file in the specified directory. We can restore it at any time with the `load` command.
+{{< /alert >}}
+
+To list all the Cloud Pods available locally or remote (LocalStack Pod's platform), you can use the `list` command:
+
+{{< command >}}
+$ localstack pod list
+
+localstack pod list
+┏━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━┓
+┃ local/remote ┃ Name            ┃
+┡━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━┩
+│ local+remote │ reproducible-ml │
+└──────────────┴─────────────────┘
+{{< / command >}}
+
+You can inspect the contents of a Cloud Pod using the `inspect` command:
+
+{{< command >}}
+$ localstack pod inspect reproducible-ml
+{{< / command >}}
+
+While you save a Cloud Pod, it is automatically published on the LocalStack Pod's platform and can be shared with other users in your organization. While saving an already existing Cloud Pod, we would create a new version, which is eventually uploaded to the LocalStack Pod's platform.
```

Comment:
Briefly discussed the naming in a PR with @giograno recently as well. I think we should either standardize around "`LocalStack Pods platform`" or simply "`LocalStack platform`". I would personally prefer the latter, as it is more generic and refers to our hosted platform as a general concept, rather than assigning it a specific product name.

```suggestion
While you save a Cloud Pod, it is automatically published on the LocalStack platform and can be shared with other users in your organization. While saving an already existing Cloud Pod, we would create a new version, which is eventually uploaded to the LocalStack platform.
```